scripts/pre-push.py

```
@@ -0,0 +1,142 @@
+    try:
+        # Check Poetry installation first
+        check_poetry_installation()
+
+        # Parse arguments
+        parser = argparse.ArgumentParser(description="Run pre-push checks")
+        parser.add_argument(
+            "--skip-tests",
+            action="store_true",
+            help="Skip running tests"
+        )
+        parsed_args = parser.parse_args(args)
```

Ruff (0.8.2)

Local variable `parsed_args` is assigned to but never used

Remove assignment to unused variable `parsed_args`

(F841)